src/components/Movies.vue

```diff
@@ -57,35 +70,35 @@ export default {
   name: "Movies",
   data() {
     return {
-      headers: [
-        {
-          sortable: false,
-          text: "#",
-          value: "movie.id"
-        },
-        {
-          sortable: false,
-          text: "Title",
-          value: "movie.title"
-        }
-      ],
       error: null,
-      search: ""
+      search: "",
+      genre: ""
     };
   },
   computed: {
     movies() {
+      if (this.genre.length > 0 && this.genre !== "all") {
```

Comment:
this part of logic should be in store, as generic getter which if argument is passed will filter by argument else return films withouth filters aplied